components/status/node/index.mjs

```diff
@@ -56,10 +56,9 @@ export const externals = {
 
 export const run = (root) => {
   const errors = [];
-
   const node_version = externals.getNodeVersion();
-  const require = createRequire(import.meta.url);
-  const versions = require("../../../package.json").engines.node;
+  const versions = parseJSON(readFile(new URL("package.json", home), "utf8"))
```

Comment:
The challenge is to have something that works both in bundles (prod) and in source files (test). I will see what I can do.

Reply:
> The challenge is to have something that works both in bundles (prod) and in source files (test). I will see what I can do.

Should be simple if you keep the same directory depth; otherwise just keep going up until you find package.json. There are npm packages for that but it's pretty trivial so it's probably not worth pulling them in.

Reply:
Yes but what do you mean by preload? Like load the file at the top-level of the module? That is easy indeed :) I thought you meant bundle the `package.json` file somehow.

Reply:
Yeah, that's what I meant :) Why read it over and over again, just load it once and keep it parsed and accessible.

Reply:
Rebased in 277ecb0f2ee156872e6573d6dd6182e4c831e93f